<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增书籍</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-character-map">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>书名*</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>原价</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>存放位置</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>出版年份</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>600</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>存放位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>原价</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版年份</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>190</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>购买日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>购买价格</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>购买日期</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>229</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>购买价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>470</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
